src/queries.ts

```diff
@@ -80,6 +80,34 @@ export async function getAllArDriveTransactionsWithin(minBlock: number, maxBlock
 	return allEdges;
 }
 
+export async function getBundledTransactions(txIds: string[]): Promise<GQLEdgeInterface[]> {
+	const allEdges: GQLEdgeInterface[] = [];
+
+	// const blockHeight = await getBlockHeight();
+
+	let hasNextPage = true;
+
+	while (hasNextPage) {
+		const query = createBundledTxsQuery(txIds);
+		const response = await sendQuery(query);
+		if (response.edges.length) {
+			console.log(`Transactions count: ${response.edges.length}`);
+			// const mostRecentTransaction = response.edges[response.edges.length - 1];
```

Comment:
can we remove theses comments?

Reply:
e8934ce85da42872a47d21bbec3bd1c3856a7d34